Cargo.toml

```diff
@@ -24,7 +24,7 @@ prost-codec = ["raft-proto/prost-codec"]
 
 # Make sure to synchronize updates with Harness.
 [dependencies]
-log = ">0.2"
+log = "0.4"
```

Comment:
I think `>0.2` includes `0.4`?

Reply:
Perhaps it is necessary for `slog-stdlog`?